security/imx_hab4/sign-file.sh

```diff
@@ -259,6 +265,6 @@ fi
 
 # Cleanup config / mod SPL
 rm ${CSF_TEMPLATE}.csf-config
-rm ${CSF_TEMPLATE}.csf-config~
+rm -f ${CSF_TEMPLATE}.csf-config~ 2> /dev/null
```

Comment:
-f shouldn't complain if the file doesn't exist

Reply:
yes I can remove the 2> /dev/null